webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -18,7 +19,23 @@
           :src="contribution.image | proxyApiUrl"
         />
       </hc-teaser-image>
+
       <ds-card>
+        <div class="blur-toggle">
+          <label for="blur-img">{{ $t('contribution.inappropriatePicture') }}</label>
+          <input type="checkbox" id="blur-img" v-model="form.blurImage" />
+          <p>
+            <a
+              href="https://support.human-connection.org/kb/faq.php?id=113"
+              target="_blank"
+              class="link"
+            >
+              {{ $t('contribution.inappropriatePictureText') }}
+              <ds-icon name="question-circle" />
+            </a>
+          </p>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-03T17:09:37Z" title="Tuesday, December 3rd 2019, 6:09:37 pm +01:00">Dec 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,14 +7,56 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
-      <hc-teaser-image :contribution="contribution" @addTeaserImage="addTeaserImage">
+      <hc-teaser-image
+        :contribution="contribution"
+        @addTeaserImage="addTeaserImage"
+        :class="{ 'images-set-blur': checkedBlur }"
+      >
         <img
           v-if="contribution"
           class="contribution-image"
           :src="contribution.image | proxyApiUrl"
         />
       </hc-teaser-image>
+      <ds-text align="right" class="blurBox">
+        <div v-show="checkedBlur">
+          <ds-button
+            v-if="contribution"
+            class="bluricon-post"
+            icon="ban"
+            primary
+            @click.prevent="unBlur"
+          ></ds-button>
+          <img
+            v-if="contribution"
```


can this duplicate `v-if` live in the parent of these two elements?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2019-12-06T05:42:52Z" title="Friday, December 6th 2019, 6:42:52 am +01:00">Dec 6, 2019</time>_

---


yes because i have to change the image without saving it, with a constante i can't do that somehow